app/SceneDelegate.h

```diff
@@ -6,9 +6,12 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "TerminalViewController.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+extern TerminalViewController *currentTerminalViewController;
```

Comment:
I don't really like this way of getting the view controller. My other ideas are way more annoying to implement though.